tests/rest/server.go

```diff
@@ -199,6 +217,66 @@ func checkClustersEndpointWrongMethods() {
 	}
 }
 
+func constructURLForResultForOrgCluster(organizationID string, clusterID string) string {
```

Comment:
btw, I'm implementing something similar in my branch for covering server with tests

```> cat server/endpoints.go
package server

import (
        "fmt"
        "regexp"
)

const (
        MainEndpoint                    = ""
        DeleteOrganizationsEndpoint     = "organizations/{organizations}"
        DeleteClustersEndpoint          = "clusters/{clusters}"
        OrganizationsEndpoint           = "organizations"
        ReportEndpoint                  = "report/{organization}/{cluster}"
        LikeRuleEndpoint                = "clusters/{cluster}/rules/{rule_id}/like"
        DislikeRuleEndpoint             = "clusters/{cluster}/rules/{rule_id}/dislike"
        ResetVoteOnRuleEndpoint         = "clusters/{cluster}/rules/{rule_id}/reset_vote"
        ClustersForOrganizationEndpoint = "organizations/{organization}/clusters"
        MetricsEndpoint                 = "metrics"
)

// MakeURLToEndpoint creates URL to endpoint, use constants from file endpoints.go
func MakeURLToEndpoint(apiPrefix, endpoint string, args ...interface{}) string {
        re := regexp.MustCompile(`\{[a-zA-Z_0-9]+\}`)
        endpoint = re.ReplaceAllString(endpoint, "%v")
        return apiPrefix + fmt.Sprintf(endpoint, args...)
}
```

so that we don't have to change endpoint it unit tests and rest api tests if we decide to change url

Reply:
yup for future PR